src/herbivoresAndCarnivores.js

```diff
@@ -1,15 +1,60 @@
 'use strict';
 
 class Animal {
-  // write your code here
+  static alive = [];
+
+  constructor(health = 100, name, hidden = false) {
+    this.health = health;
+    this.name = name;
+    this.hidden = hidden;
+    Animal.alive.push(this);
+  }
+
+  die() {
```

Comment:
i think checkDead() would be a better naming, like, die() implies that it's making the animal die

you may want to read these [short naming conventions](https://www.syncfusion.com/blogs/post/top-javascript-naming-convention)